docs/api/presenter.md

```diff
@@ -41,7 +41,7 @@ class PlanetsPresenter extends Presenter {
 
   getModel (props, state) {
     return {
-      planets: data => data.planets
+      planets: state => state.planets
```

Comment:
Sorry... responded too soon. We have a pickle here. getModel accepts presenter props and state as arguments, however we also have the first argument of model bindings, which is repo state.

Everywhere else, we're probably using `state => state.planets`. What are your thoughts on how we can make this less confusing?